src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/GeneralOutput/WarningsSection/SearchGroupDuplicate/searchGroupDuplicate.epics.js

```diff
@@ -88,53 +75,15 @@ export const loadSearchGroupDuplicatesForReviewEpic = (action$: InputObservable,
             };
-
-            return getApi()
-                .get('trackedEntityInstances', queryParams)
-                .then((response) => {
-                    const formFoundation = getFormMetadata(programId, tetId);
-                    const pager = (response && response.pager) || {};
-                    const paginationData = !isChangePage ? {
-                        rowsCount: pager.total,
-                        rowsPerPage: pager.pageSize,
-                        currentPage: pager.page,
-                    } : null;
-
-                    const teInstances = (response && response.trackedEntityInstances) || [];
-                    const convertedInstances = teInstances
-                        .map((instance) => {
-                            const attributes = instance.attributes || [];
-
-                            const values = attributes
-                                .reduce((acc, container) => {
-                                    acc[container.attribute] = container.value;
-                                    return acc;
-                                }, {});
-
-                            const convertedValues = formFoundation.convertValues(values, convertServerToClient);
-
-                            return {
-                                id: instance.trackedEntityInstance,
-                                values: {
-                                    ...convertedValues,
-                                    /* $FlowFixMe[prop-missing] automated
-                                     * comment */
-                                    registrationDate: convertServerToClient(instance.created, dataElementTypes.DATETIME),
-                                    /* $FlowFixMe[prop-missing] automated
-                                     * comment */
-                                    registrationUnit: convertServerToClient(instance.orgUnit, dataElementTypes.TEXT),
-                                    /* $FlowFixMe[prop-missing] automated
-                                     * comment */
-                                    inactive: convertServerToClient(instance.inactive, dataElementTypes.BOOLEAN),
-                                },
-                            };
-                        });
-
-                    return duplicatesForReviewRetrievalSuccess(
-                        convertedInstances,
-                        paginationData,
-                    );
-                })
-                .catch(() =>
-                    duplicatesForReviewRetrievalFailed());
-        }));
+            const attributes = contextParam.program ?
+                getTrackerProgramThrowIfNotFound(contextParam.program).attributes :
+                getTrackedEntityTypeThrowIfNotFound((contextParam.trackedEntityType)).attributes;
+
+            return from(getTrackedEntityInstances(queryArgs, attributes)).pipe(
+                map(({ trackedEntityInstanceContainers: searchResults, pagingData }) =>
```

Comment:
I thought reusing this here since we basically perform the same request. Let me know if you think of any drawbacks please.